lib/services/database.go

```diff
@@ -686,20 +675,44 @@ func NewDatabaseFromRDSV2Cluster(cluster *rdsTypesV2.DBCluster) (types.Database,
 		})
 }
 
+func rdsSubnetGroupToNetworkInfo(subnetGroup *rdsTypesV2.DBSubnetGroup) (vpcID string, subnets []string) {
+	if subnetGroup != nil {
```

Comment:
```suggestion
	if subnetGroup == nil {
		return
	}
```

Similar idea to https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow, as in this reduces the indents.